app/docker/views/services/create/createservice.html

```diff
@@ -63,7 +63,7 @@
                 </div>
                 <!-- !host-port -->
                 <span style="margin: 0 10px 0 10px">
-                  <i class="fa fa-long-arrow-alt-right" aria-hidden="true"></i>
+                  <pr-icon icon="'svg-arrowright'"></pr-icon>
```

Comment:
any reason not to use feather's arrow-right icon?

Reply:
I got it as the replacement from https://portainer-style-guide.webflow.io/

![image](https://user-images.githubusercontent.com/83188384/198133870-9a3f9821-f8b9-4487-9e04-bbc2c5bf2aa9.png)

I'll rename this svg to arrowrightlong to be more obvious how it is different from feather